### ISSUE TITLE
Playlist

### ISSUE THREAD

**Issue by akhilleusuggo** [WWBN/AVideo#3024]
Under Playlist , could you add an option for the channel owner to '' Add new playlist ''
![image](https://user-images.githubusercontent.com/52116577/81409251-74698780-9147-11ea-9af2-9fdd11c1561c.png)

**Comment by DanielnetoDotCom** [comment]
That is not a bad idea, but the thing is...

to create and name it is fine, but what about select the videos to add in the playlist?

**Comment by akhilleusuggo** [comment]
From my videos : 
![image](https://user-images.githubusercontent.com/52116577/81431058-30877a00-9169-11ea-9d6d-b1eab59d098e.png)

Also you could add it on Video's section '' Create playlist/program ''

**Comment by DanielnetoDotCom** [comment]
Yes, maybe in the videos section only is better, as long as we can add videos from there.
